source/Octopus.Tentacle/Scripts/NoopScriptStateWorkspace.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using Octopus.Tentacle.Contracts;
+
+namespace Octopus.Tentacle.Scripts
+{
+    public class NoopScriptStateWorkspace : IScriptStateWorkspace
+    {
+        public bool StateExists => false;
+        public ScriptState Create()
```

Comment:
Maybe Create doesn't need to return a ScriptState, that would make the no-op implementation just do nothing.